contracts/libraries/Validator.sol

```diff
@@ -72,4 +31,83 @@ library Validator {
         }
         return computedHash == rootHash;
     }
+
+    // @param txHash      transaction hash
+    // @param rootHash    block header of the merkle tree
+    // @param input0      indicator for nonzero first input
+    // @param input1      indicator for nonzero second input
+    // @param sigs        transaction signatures
+    function checkSigs(bytes32 txHash, bytes32 confirmationHash, bool input0, bool input1, bytes sigs, bytes confirmSignatures)
+        internal
+        pure
+        returns (bool)
+    {
+        require(sigs.length == 130, "two transaction signatures required");
```

Comment:
In one input transaction only one signature is required. Can you use input1 when doing this check?

Reply:
Yes we could do that. I was thinking that even if there is only 1 input, the user should just pad the signature with 65 0 bytes.

Now that I think about it, I think it's better to allow only passing in 1 sig/confirmsig